linkis-engineconn-plugins/spark/src/main/resources/python/mix_pyspark.py

```diff
@@ -118,6 +118,7 @@ def isImportAllPackageUnderSparkSql(self):
 java_import(gateway.jvm, "org.apache.spark.api.java.*")
 java_import(gateway.jvm, "org.apache.spark.api.python.*")
 java_import(gateway.jvm, "org.apache.spark.mllib.api.python.*")
+java_import(gateway.jvm, "org.apache.spark.sql.api.python.*")
 
```

Comment:
`org.apache.spark.sql.api.python`
 new pyspark version rename package?

Reply:
The package was not renamed, it always exist. but a new method was added, and this method reported an error of `'JavaPackage' object is not callable`, so import this package to solve this problem